iOS/Flows/Conversation/Cells/MessageSubcell.swift

```diff
@@ -25,25 +26,37 @@ class MessageSubcell: UICollectionViewCell {
 
     private func initializeViews() {
         self.contentView.addSubview(self.content)
+        self.contentView.addSubview(self.detailView)
     }
 
     override func layoutSubviews() {
         super.layoutSubviews()
 
-        self.content.expandToSuperviewSize()
+        self.detailView.expandToSuperviewWidth()
+        self.content.expandToSuperviewWidth()
+
+        self.content.height = self.bounds.height - (self.detailView.height - (self.content.bubbleView.tailLength - Theme.ContentOffset.short.value))
```

Comment:
We should add a comment about why we're using bounds instead of the frame height.

Reply:
agreed